care/users/api/viewsets/users.py

```diff
@@ -70,6 +71,10 @@ def get_user_type(
     user_type = filters.CharFilter(method="get_user_type", field_name="user_type")
 
 
+class UserFacilitiesPagination(LimitOffsetPagination):
+    page_size = 20
+
+
```

Comment:
```suggestion
```
let's use the default pagination class. We can always specify the page size the fe wants

Reply:
Done